src/Schema/Tables/UserData.sql

```diff
@@ -0,0 +1,16 @@
+﻿CREATE TABLE [dbo].[UserData]
+(
+	[UserId] INT NOT NULL IDENTITY(1, 1), 
+    [FirstName] NVARCHAR(100) NOT NULL, 
+    [LastName] NVARCHAR(100) NOT NULL, 
+    [BirthDate] DATE NOT NULL, 
+    [PhoneNumber] VARCHAR(12) NOT NULL, 
```

Comment:
According to WIKI numbers can be up to 15 digits. But make sure if you want some more readable formats like 
+38-067-378-37-43  - only such Ukrainian number is 18 characters long. Think about what is the best solution.